### DISCUSSION TITLE
Installing composer on a system where an old version is installed

### DISCUSSION THREAD

**Discussion by baltusf** [composer/composer#10553] [Q&A / Support]
Hi, can someone help me? My hosting provider has via cpanel an old version of composer and drush installed. (1.8.0)
I did downlaod the new version several times, but when i type in composer the old version is showed. I can't update the installed version, no permission. Can someone help me so i can install composer (and also drush) in my root? And by typing composer it runs that one.?

I will be pleased for a solution, so i can update also drupal with composer.
Thanks,
Fred

**Comment by Seldaek** [comment]
You may have to execute it as `php /path/to/composer.phar`  instead if you can't replace the one installed globally. Or move it to some dir that you make sure is in your PATH env var *before* the global old one.
